extensions/interactions/legacy_interaction_specs_by_state_version/interaction_specs_state_v41.json

```diff
@@ -200,7 +200,7 @@
     "id": "ItemSelectionInput",
     "customization_arg_specs": [
       {
-        "description": "Minimum number of selections permitted.",
+        "description": "Minimum number of selections permitted",
```

Comment:
I don't think you need to change the legacy files.

Reply:
@iamprayush I have unchange the legacy files and it work fine

Thanks!